phylotoast/util.py

```diff
@@ -63,37 +60,35 @@ def parseFASTA(fastaFNH):
         if line[0] == ">":
             # conclude previous record
             if seq:
-                recs.append(FASTARecord(ID, descr, "".join(seq)))
+                recs.append(FASTARecord(seqID, descr, "".join(seq)))
                 seq = []
             # start new record
-            line = line[1:].split()
-            ID, descr = line[0], " ".join(line[1:])
+            line = line[1:].split(None, 1)
```

Comment:
keyword arg aren't accepted in str.split() method. valid syntax are - 
1. line = line[1:].split(None, 1)
2. line = line[1:].split(" ", 1)

adding a comment could be a third way